data/helpers/projectRouter.js

```diff
@@ -0,0 +1,146 @@
+const express = require('express');
+
+const Project = require('./projectModel');
+// const Action = require('../helpers/actionModel');
+
+const router = express.Router();
+
+// Project CRUD
+
+router.get('/', async(req, res) => {
+    try {
+        console.log(res);
+        // const project = await Project.get(req.query);
+        const project = await Project.get();
+        res.status(200).json(project);
+    } catch (error) {
+        console.log(error);
+        res.status(500).json({ message: 'Error retrieving the projects' });
+    }
+});
+
+router.get('/:id', async (req, res) => {
```

Comment:
Here you have correctly implemented the two get endpoints, also, nice use of ternary operators.